routers/user/auth.go

```diff
@@ -1433,3 +1434,17 @@ func MustChangePasswordPost(ctx *context.Context, cpt *captcha.Captcha, form aut
 
 	ctx.Redirect(setting.AppSubURL + "/")
 }
+
+// BuildPasswordComplexityError builds the error message when password complexity checks fail
+func BuildPasswordComplexityError(ctx *context.Context) string {
+	var buffer bytes.Buffer
+	buffer.WriteString(ctx.Tr("form.password_complexity"))
+	buffer.WriteString(`<ul>`)
+	for _, c := range password.GetActiveComplexities() {
+		buffer.WriteString("<li>")
+		buffer.WriteString(ctx.Tr("form." + c.TrNameOne))
```

Comment:
Rather than prefix with `form.` I would say to add the `form.` to the `TrNameOne` to keep the translation key in one spot in the code.

Reply:
> 
> 
> Rather than prefix with `form.` I would say to add the `form.` to the `TrNameOne` to keep the translation key in one spot in the code.

@jolheiser You're right. Fixed.